fastlane/lib/fastlane/actions/docs/upload_to_play_store.md

```diff
@@ -83,6 +83,18 @@ Expansion files (obbs) found under the same directory as your APK will also be u
 - they are identified as type 'main' or 'patch' (by containing 'main' or 'patch' in their file name)
 - you have at most one of each type
 
+If you want only update expansion files from previous version on Google Play use
```

Comment:
```suggestion
If you _only_ want to update the expansion files on Google Play use
```

What did the `from previous version ` refer to here? Are you updating the previous version to a new one or are you uploading obb files from a previous version?

Reply:
It means user is uploading new APK without obb file but links this new APK to old obb (already uploaded in past with older APK).

Reply:
So this would describe the situation then?

```suggestion
If you only want to update the APK, but keep the expansion files from the previous version on Google Play use
```